config/jenkins-jobs_infra.ci.jenkins.io.yaml

```diff
@@ -509,11 +509,22 @@ jobsDefinition:
             description: "docs.jenkins.io File Share Service Principal Writer"
             subscriptionId: "${JENKINSINFRA_AZURE_PRIMARY_SUBSCRIPTION_ID}"
             tenant: "${JENKINSINFRA_AZURE_TENANT_ID}"
+      stats.jenkins.io_previews:
```

Comment:
```suggestion
      stats.jenkins.io:
```

Rationale:
- Less JobDSL changes (see current helmfile diff)
- Namespaced by the parent job
- better github checks names (ref. https://github.com/jenkins-infra/helm-charts/blob/d7837630e659b10a4bc0b145804c31da3349ebfc/charts/jenkins-jobs/templates/_helpers.tpl#L133)

Reply:
Without changes to jenkins-jobs helm chart I will have to anyway, this name is reused as repository name, my change would have needed to be more complete to be valid cf helmfile diff:

<img width="789" alt="image" src="https://github.com/jenkins-infra/kubernetes-management/assets/91831478/0dc0ea95-6e78-48b6-97be-6f7c01b22483">

Reply:
> more complete

https://github.com/jenkins-infra/helm-charts/blob/d7837630e659b10a4bc0b145804c31da3349ebfc/charts/jenkins-jobs/templates/_helpers.tpl#L112